workerd-vite-utils/src/workerdBootloader.js.txt

```diff
@@ -22,30 +22,37 @@ async function __vite_ssr_import__(moduleId) {
 		return __ourPrivateModuleRegistry__.get(moduleId);
 	}
 
+	if (moduleId === "seroval") {
+		moduleId =
+			"/Users/jculveyhouse/cf-repos/vite-workerd-ssr-handler-experimentation/node_modules/.pnpm/node_modules/seroval";
+	}
```

Comment:
this would be best to get fixed soon, it doesn't need to block the merge (since we're just experimenting)

Reply:
as discussed let's keep it as is, not ideal but we can just hack around more before dealing with this 🙂